pybess/bess.py

```diff
@@ -495,6 +495,15 @@ def _configure_gate_hook(self, hook, module,
         request.arg.Pack(arg)
         return self._request('ConfigureGateHook', request)
 
+    def _configure_resume_hook(self, hook, arg, enable=None):
```

Comment:
Since the method is meant to be used externally, the method name should not begin with `_`.

(`enable=True` should be fine...?)

Reply:
Ah, yup-- but I think we still need the `is None` check below since the bessctl command parser sets unpresent args to `None`